pkgs/by-name/pu/pulsar/update.mjs

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env nix-shell
 /*
-#!nix-shell -i node -p nodejs_18
+#!nix-shell -i node -p nodejs
```

Comment:
I decided to go straight to LTS because there's nothing in this script that's not using very stable Node.js features, I suspect nodejs_18 was picked because it was the LTS at the time of https://github.com/NixOS/nixpkgs/commit/aa4cf231c4d1d062dd5e91d5d7807da4442c4f5f, so to lower maintenance burden when nodejs_20 gets retired, let's make it target latest LTS instead.